_targets.R

```diff
@@ -19,7 +19,7 @@ tar_option_set(
   memory = "persistent" # default option
 )
 
-years_to_run <- c("1920")
+years_to_run <- c("1819")
```

Comment:
Just a reversion as it isn't a meaningful change.
```suggestion
years_to_run <- c("1920")
```